docker-compose.yml

```diff
@@ -1,40 +1,29 @@
 version: '3.7'
-services:
-  offline:
-    image: aligent/serverless:offline
-    networks:
-      - localstack-net
-    command: tail -f /dev/null
-    volumes:
-      - .:/app
```

Comment:
Why are we no longer mounting these?

Reply:
My `docker-compose.yml` file will only spin up localstack container. Therefore, it does not need to mount `/app` folder. This folder will be mounted to another container (the aligent/serverless:latest one) as normal.